Web/Presenters/GroupPresenter.php

```diff
@@ -170,6 +170,11 @@ function renderEdit(int $id): void
             $club->setShortcode(empty($this->postParam("shortcode")) ? NULL : $this->postParam("shortcode"));
 	        $club->setWall(empty($this->postParam("wall")) ? 0 : 1);
             $club->setAdministrators_List_Display(empty($this->postParam("administrators_list_display")) ? 0 : $this->postParam("administrators_list_display"));
+            if(strpos($this->postParam("website") ?? "", "https://") === 0 || strpos($this->postParam("website") ?? "", "http://") === 0) {
+                $club->setWebsite(empty($this->postParam("website")) ? NULL : $this->postParam("website"));
+            } else {
+                $club->setWebsite(empty($this->postParam("website")) ? NULL : "http://" . $this->postParam("website"));
```

Comment:
Parse the protocol (secure or not) in dedicated variable

Reply:
Done

Reply:
Oh, I didn't notice the new message